### DISCUSSION TITLE
Transition Provisions for Registered VASPs and In Process Applications - Sec 6(A)

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#44] [General]
The thirty-day period for existing trading platform or custodians to submit a license application upon commencement of the VASP Act amendments is insufficient. The application requirements in the proposed VASP Regulations are comprehensive and include 19 deliverables. In order to prepare a high-quality license application, it will take a minimum of ninety-days even if preparations are made prior to commencement of the Act.

A transition approach will also need to be developed for trading platforms and custodians that have in-progress registration applications. Ideally all registration applications for trading platforms and custodians will be processed prior to the commencement of the Act but it would be inefficient and unfair if applicants that have been subjected to the registration application process are not granted registration and a grace period to submit their licensing application. I recommend that all registration applications submitted prior to the commencement of the Act be processed and provided with a 90 days period to submit their licensing application upon registration approval.

**Comment by AGIAI** [comment]
Agreed

**Comment by Clearwaterscrypto** [comment]
Having 30 days to submit an application will result in CIMA receiving poor quality applications which will be counterproductive, creating more work for the regulator and result in significant delays to the processing times. Agreed that 90 days is reasonable and will deliver better results all round.

**Comment by Cayman-Blockahain-Association** [comment]
Based on the BACI roundtable discussion held with our members the consensus was that 30 days to submit applications would be too short and 90 days would be more reasonable.

**Comment by CaraProvenance** [comment]
Agreed and if possible, it would be favorable for all registrants/existing platforms to be advised in advance of the commencement of the expectation, as a notification via the CIMA connect portal.

**Comment by lkoseland** [comment]
Agree. A grandfathering provision should be inserted for those applications currently in process at the time of commencement of the Virtual Asset (Service Providers) (Amendment) Act, 2024.
